PIVX-Labs/MyPIVXWallet
# Dockerbuild

## Abstract
Add a docker system to host a test version of our the MPW infrastructure. As of right now, no blocks are created, and no PIVs are present on the regtest, but can be createding by calling the `generate` RPC on the pivx node.

depends on #287 

---

## Testing
To test this PR, it's suggested to attempt these user flows, or variations of these:
- Run `docker compose build` and test that the build works as expected.
- Run `docker compose up`
- Open a browser tab on `localhost:8080` and test that MPW is able to talk with the local explorer and RPC node
